clients/imodelhub/src/imodelhub/iModels.ts

```diff
@@ -399,87 +390,70 @@ export class IModelsHandler {
+    const imodels = await this._handler.getInstances<HubIModel>(requestContext, HubIModel, this.getRelativeUrl(iTwinId, query.getId()), query.getQueryOptions());
+    Logger.logInfo(loggerCategory, `Finished querying iModels in context`, () => ({ contextId: iTwinId, count: imodels.length }));
 
     return imodels;
   }
 
-  // SWB
-  /** Delete an iModel with specified id from a context. This method is not supported in iModelBank.
+  /** Delete an iModel with specified id from its parent iTwin. This method is not supported in iModelBank.
    * @param requestContext The client request context.
-   // SWB
-   * @param contextId Id for the iModel's context. For iModelHub it should be the id of the iTwin context ([[ITwin]]).
+  // SWB When is the second part of the comment not true?
+   * @param iTwinId Id for the iModel's parent iTwin. For iModelHub it should be the id of the iTwin ([[ITwin]]).
    * @param iModelId Id of the iModel to be deleted. See [[HubIModel]].
    * @throws [[IModelHubError]] with [IModelHubStatus.iModelDoesNotExist]$(bentley) if iModel with specified id does not exist.
    * @throws [[IModelHubError]] with [IModelHubStatus.UserDoesNotHavePermission]($bentley) if the user does not have DeleteiModel permission.
    * @throws [Common iModelHub errors]($docs/learning/iModelHub/CommonErrors)
    */
-  // SWB
-  public async delete(requestContext: AuthorizedClientRequestContext, contextId: string, iModelId: GuidString): Promise<void> {
-    requestContext.enter();
-    Logger.logInfo(loggerCategory, "Started deleting iModel", () => ({ iModelId, contextId }));
+  public async delete(requestContext: AuthorizedClientRequestContext, iTwinId: string, iModelId: GuidString): Promise<void> {
+    Logger.logInfo(loggerCategory, "Started deleting iModel", () => ({ iModelId, contextId: iTwinId }));
     ArgumentCheck.defined("requestContext", requestContext);
-    // SWB
-    ArgumentCheck.validGuid("contextId", contextId);
+    ArgumentCheck.validGuid("iTwinId", iTwinId);
     ArgumentCheck.validGuid("iModelId", iModelId);
 
     if (this._handler.getCustomRequestOptions().isSet) {
       // to add custom request options, request with body is needed.
       const imodel = new HubIModel();
       imodel.id = iModelId;
       imodel.changeState = "deleted";
-      await this._handler.deleteInstance(requestContext, this.getRelativeUrl(contextId, iModelId), imodel);
+      await this._handler.deleteInstance(requestContext, this.getRelativeUrl(iTwinId, iModelId), imodel);
     } else {
-      await this._handler.delete(requestContext, this.getRelativeUrl(contextId, iModelId));
+      await this._handler.delete(requestContext, this.getRelativeUrl(iTwinId, iModelId));
     }
-    requestContext.enter();
-    Logger.logInfo(loggerCategory, "Finished deleting iModel", () => ({ iModelId, contextId }));
+    Logger.logInfo(loggerCategory, "Finished deleting iModel", () => ({ iModelId, contextId: iTwinId }));

```

Comment:
Why does this say context?

Reply:
Removed questions from clients since they will be removed soon. Marking this as resolved.